reconcile/queries.py

```diff
@@ -320,15 +320,50 @@ def get_integrations(managed=False):
       name
       keep_hours
     }
+    {% if work_fleets %}
+    workerFleets {
+      account
+      identifier
+      credentialsId
+      fsRoot
+      labelString
+      numExecutors
+      idleMinutes
+      minSpareSize
+      noDelayProvision
+      namespace{
+        name
+        managedExternalResources
+        externalResources {
+          provider
+          ... on NamespaceTerraformProviderResourceAWS_v1 {
+            provisioner {
+              name
+              resourcesDefaultRegion
+            }
+            resources {
+              provider
+              ... on NamespaceTerraformResourceASG_v1 {
+                identifier
+                defaults
+                overrides
+              }
+            }
+          }
+        }
+      }
+    }
+    {% endif %}
   }
 }
 """
 
 
-def get_jenkins_instances():
+def get_jenkins_instances(work_fleets=False):
```

Comment:
nit: shouldn't this be called `worker_fleets`?

```suggestion
def get_jenkins_instances(worker_fleets=False):
```